web/components/Navbar.tsx

```diff
@@ -187,20 +189,48 @@ export default function Navbar() {
           spacing={10}
         >
           <Link as={NextLink} href={Pages.FEED}>
-            <Text>Feed</Text>
+            <Text
+              color={currPage === "FEED" ? "#57A0D5" : "black"}
```

Comment:
Do something like `router.pathName === Pages.FEED`. Use Chakra's semantic tokens that we have defined whenever you can - I think it's `"text-primary"`

Reply:
Also, just noticed this, but make sure you change the hover underline color to be the same as the text color

Reply:
whoops I totally did not see that, will do